### PR TITLE
fix(composer): update overlay and settings UI

### PR DESCRIPTION
**Pull request by sheilaXu**
## Overview
Merge https://github.com/awslabs/iot-app-kit/pull/960 to main
- update overlay and settings UI based on latest UX mock change

### Scene Composer
For `scene-composer` package changes specifically, you can preview the component in the published storybook artifact. To do this, wait for the `Publish Storybook` action to complete below.

- Click on the workflow details
- Select the Summary item on the left
- Download the zip file

To run the storybook build locally, you need a local static web server:

```
npm install -g httpserver
cd <Extracted Zip Directory>
httpserver
```

Then open the website http://localhost:8080 to run the doc site.

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/scene-composer/src/components/panels/SettingsPanel.tsx [hunk 1]
```diff
@@ -100,26 +101,41 @@ export const SettingsPanel: React.FC<SettingsPanelProps> = ({ valueDataBindingPr
...
-        />
-        <ComponentVisibilityToggle
-          componentType={KnownComponentType.Tag}
-          label={intl.formatMessage(visibilityToggleLabels[KnownComponentType.Tag])}
-        />
-        {overlayEnabled && (
+        <FormField
+          label={
+            <Box fontWeight='bold'>
+              {intl.formatMessage({ defaultMessage: 'Toggle visibility', description: 'FormField label' })}
```
Comment (tracy-french) [inline packages/scene-composer/src/components/panels/SettingsPanel.tsx:107]: Nit: Shouldn't `<FormattedMessage />` be used instead here?

File: packages/scene-composer/src/components/panels/SettingsPanel.tsx [hunk 2]
```diff
@@ -100,26 +101,41 @@ export const SettingsPanel: React.FC<SettingsPanelProps> = ({ valueDataBindingPr
...
+              />
+            </>
+          )}
+          {overlayEnabled && (
+            <>
+              <Divider />
+              <ComponentVisibilityToggle
+                componentType={Component.DataOverlaySubType.TextAnnotation}
+                label={intl.formatMessage(visibilityToggleLabels[Component.DataOverlaySubType.TextAnnotation])}
+              />
```
Comment (tracy-french) [inline packages/scene-composer/src/components/panels/SettingsPanel.tsx:135]: Nit: Could you ensure there are descriptions for all translations to make them easier to find?

**Review by tracy-french — APPROVED** [review]
Thank you for your work!

**Review by mumanity — APPROVED** [review]

File: packages/scene-composer/src/components/panels/SettingsPanel.tsx (hunk 2 above)
Comment (sheilaXu) [reply packages/scene-composer/src/components/panels/SettingsPanel.tsx:135]: yeah, descriptions are in `visibilityToggleLabels`

**Pull request merged**